tests/integration/reconcile/test_compare.py

```diff
@@ -113,15 +113,15 @@ def test_compare_data_for_report_hash(mock_spark, tmp_path: Path):
 def test_capture_mismatch_data_and_cols(mock_spark):
     source = mock_spark.createDataFrame(
         [
-            Row(s_suppkey=1, s_nationkey=11, s_name='supp-1', s_address='a-1', s_phone='ph-1', s_acctbal=100),
+            # Row(s_suppkey=1, s_nationkey=11, s_name='supp-1', s_address='a-1', s_phone='ph-1', s_acctbal=100), --we don't expect this row in the source
             Row(s_suppkey=2, s_nationkey=22, s_name='supp-22', s_address='a-2', s_phone='ph-2', s_acctbal=200),
             Row(s_suppkey=3, s_nationkey=33, s_name='supp-3', s_address='a-3', s_phone='ph-3', s_acctbal=300),
             Row(s_suppkey=5, s_nationkey=55, s_name='supp-5', s_address='a-5', s_phone='ph-5', s_acctbal=400),
         ]
     )
     target = mock_spark.createDataFrame(
         [
-            Row(s_suppkey=1, s_nationkey=11, s_name='supp-1', s_address='a-1', s_phone='ph-1', s_acctbal=100),
+            # Row(s_suppkey=1, s_nationkey=11, s_name='supp-1', s_address='a-1', s_phone='ph-1', s_acctbal=100), --we don't expect this row in the source
```

Comment:
can you add a comment here, saying these dataframe contain only mismatch information so removed the matching row.

Reply:
but if there is matching row then what is the behaviour create a separate tests for that. False Postiives.